website/tosti/settings/base.py

```diff
@@ -28,14 +28,20 @@
     "django.contrib.staticfiles",
     "admin_auto_filters",
     "import_export",
+    "guardian",
     "marietje",
     "users",
     "tosti",
     "venues",
     "orders",
 ]
 
-AUTH_USER_MODEL = "users.User"
```

Comment:
Changing this line will make the project not use our user model anymore, that is why we can't overwrite the __str__ and short name and why we are not seeing the normal names but the usernames in the template, we might want to change this to use our User model again.